app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt

```diff
@@ -6,5 +6,6 @@ import javax.inject.Inject
 class DebugModule @Inject constructor() {
     fun sendCryingProbabilityEvent(cryingProbability: Float) {}
     fun sendNotificationEvent(notificationInformation: String) {}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/89ff9ffb603a83b79e177dd00519720c6a8d52ac/app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugModule.kt#L8"></span>Function parameter notificationInformation is unused.</td>
    </tr>
  </tbody>
</table>